Project.toml

```diff
@@ -5,12 +5,10 @@ version = "0.1.2"
 
 [deps]
 CEnum = "fa961155-64e5-5f13-b03f-caf6b980ea82"
-ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
```

Comment:
We still need `FixedPointNumbers` and `Colors` + the [transpose definitions from ImageCore](https://github.com/JuliaImages/ImageCore.jl/blob/65b42543fc90e676eb306db5c0d642b1d8a7617f/src/ImageCore.jl#L152).